### ISSUE TITLE
Ensure docker release containers contain proper ca certs

### ISSUE THREAD

**Issue by LucioFranco** [timberio/vector#1005]
Currently, at least our debian container does not contain updated ca-certs I was able to produce ssl errors while trying to connect to aws. We need to ensure that these containers are able to talk securely.

**Comment by binarylogic** [comment]
Please

- Provide more detail on what you did to produce this.
- Any logs or output you experienced.
- Add a note that states this should be tested in CI in some way to ensure we don't regress.

**Comment by LucioFranco** [comment]
@binarylogic we just need to ensure `update-ca-certificates` (this is the one for debian, I have not tested alpine) has been run on the containers I believe. @a-rodin may have a better idea.

**Comment by linc01n** [comment]
I have the same problem with the ca-certificates while connecting to Amazon Elasticsearch Service as sink. I need to install ca-certificates on top of the official debian docker image. 

This issue is quite hard to test. The slim image missing `wget`,`curl`, `openssl` it took me a while to hunt down it is a system ca issue.

**Comment by a-rodin** [comment]
@LucioFranco It seems like if we don't need to add self-signed certificates there, just installing `ca-certificates` package should be enough as [this answer](https://askubuntu.com/questions/645818/how-to-install-certificates-for-command-line/859887#859887) suggests. I've tested installing `ca-certificates` packages and `http` sink with SSL and it worked fine.

**Comment by LucioFranco** [comment]
@a-rodin Agreed, does alpine have a similar command?

**Comment by a-rodin** [comment]
@LucioFranco I believe the packages called [ca-certificates](https://pkgs.alpinelinux.org/package/v3.3/main/x86/ca-certificates) too. So the command would be

```
apk update && apk add ca-certificates && rm -rf /var/cache/apk/*
```

**Comment by binarylogic** [comment]
I believe this was fixed in #1014, closing.
